modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/RelationshipObjectFieldBusinessType.java

```diff
@@ -164,6 +209,23 @@ private ObjectDefinition _getObjectDefinition(ObjectField objectField)
 			objectRelationship.getObjectDefinitionId1());
 	}
 
+	private ObjectDefinition _getObjectDefinition(
```

Comment:
I think this method is not necessary since it is only used in one place in the code and could be substituted by passing the `objectDefinitionId` as argument